pkg/build/generator/generator.go

```diff
@@ -508,9 +526,9 @@ func (g *BuildGenerator) resolveImageSecret(ctx kapi.Context, secrets []kapi.Sec
 }
 
 // getNextBuildName returns name of the next build and increments BuildConfig's LastVersion.
-func getNextBuildName(bc *buildapi.BuildConfig) string {
-	bc.Status.LastVersion++
-	return fmt.Sprintf("%s-%d", bc.Name, bc.Status.LastVersion)
+func getNextBuildName(buildConfig *buildapi.BuildConfig) string {
+	buildConfig.Status.LastVersion++
+	return fmt.Sprintf("%s-%d", buildConfig.Name, buildConfig.Status.LastVersion)
```

Comment:
absolutely nothing.  i think i wrote some new code using buildConfiguration and then that lead to refactoring and i just started changing it.  fundamentally i think it reads better, though, especially for someone not used to working in this code.